### DISCUSSION TITLE
Slack link on https://www.serverless.com/support is not functional.

### DISCUSSION THREAD

**Discussion by bgshacklett** [serverless/serverless#10912] [General]
Hello,

I was looking for a Slack workspace for Serverless, and though I had found one, but it appears that the link at https://www.serverless.com/support is not functional. I get an error stating: "This link is no longer active" when I click on it.

Is there any plan to update the link, or might it make sense to remove it entirely?

**Comment by mnapoli** [comment]
@bgshacklett thanks for reporting it, I've forwarded this internally to solve this!

In the meantime you can use that link: https://join.slack.com/t/serverless-contrib/shared_invite/zt-13unxiht3-wTSFk910WilkYNpECmlPDA

**Reply by bgshacklett** [reply]
That worked. Thanks!

**Reply by mcopik** [reply]
@mnapoli Neither of those links work anymore: "Slack This link is no longer active".

**Reply by austencollins** [reply]
@mcopik Thanks for the heads up. Just fixed it on the top of the [README](https://github.com/serverless/serverless)

**Reply by brettdh** [reply]
@austencollins and now that link doesn't work anymore either. Is there a permanent solution possible?

**Reply by brettdh** [reply]
The Slack link in the dashboard is also expired.

**Reply by walidelnozahy** [reply]
@brettdh Hey thanks for the heads up, I've addressed this issue both on the site and in the [README](https://github.com/serverless/serverless). You should now be able to join the community slack from https://serverless.com/slack.

**Reply by sharathsoma** [reply]
@walidelnozahy The link on the Readme seems to be expired too, do we have a new link to join the channel please ?
